devices/common_patches/h7_exti_dualcore.yaml

```diff
@@ -0,0 +1,853 @@
+          bitOffset: 14
+          bitWidth: 1
+        MR15:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 15
+          bitWidth: 1
+        MR16:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 16
+          bitWidth: 1
+        MR17:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 17
+          bitWidth: 1
+        MR18:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 18
+          bitWidth: 1
+        MR19:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 19
+          bitWidth: 1
+        MR20:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 20
+          bitWidth: 1
+        MR21:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 21
+          bitWidth: 1
+        MR22:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 22
+          bitWidth: 1
+        MR23:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 23
+          bitWidth: 1
+        MR24:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 24
+          bitWidth: 1
+        MR25:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 25
+          bitWidth: 1
+        MR26:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 26
+          bitWidth: 1
+        MR27:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 27
+          bitWidth: 1
+        MR28:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 28
+          bitWidth: 1
+        MR29:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 29
+          bitWidth: 1
+        MR30:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 30
+          bitWidth: 1
+        MR31:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
+          bitOffset: 31
+          bitWidth: 1
+      C1EMR1:
+        description: EXTI event mask register
+        addressOffset: 0x00C4
+        resetValue: 0x00000000
+        fields:
+        MR0:
```

Comment:
Indentation is wrong here so I don't think it will do what you want...

Reply:
Fixed, along with register names